server/src/api/events/event.service.ts

```diff
@@ -243,16 +326,16 @@ export const createEvent = async (params: any): Promise<response> => {
             ($1, $2, $3, true, $4, $5)
           RETURNING *;`,
     values: [
-      params.seasonid_fk,
+      params.seasonIdFk,
```

Comment:
Can we leave this as `seasonid_fk` to keep it consistent with the db table?

Reply:
I changed it due to ESlint complaining about camel case notation.  I can change it back but we will get linting errors.